src/GovUk.Education.ExploreEducationStatistics.Common/Utils/Html/HtmlToTextConverter.cs

```diff
@@ -181,17 +181,21 @@ private void ParseListElement(IElement element)
             var isOrdered = element.LocalName == "ol";
             // Add indentation to align list item children
             // with where the bullet/number ends.
-            var indentation = isOrdered ? "   " : "  ";
+            var baseIndent = isOrdered ? 2 : 1;
 
             var listItems = element.ChildNodes
-                .Where(node => node is IElement { LocalName: "li" });
+                .Where(node => node is IElement { LocalName: "li" })
+                .ToList();
 
             ParseNodes(
                 listItems,
                 (item, index) =>
                 {
                     _builder.Append(isOrdered ? $"{index + 1}. " : "- ");
```

Comment:
Maybe using this string to determine the size of `indent` rather than having to calculate it based off `baseIndent` and `(index + 1).ToString().Length`.  Seeing as you already have this string constructed I guess you could just do something like:

```
var lineItem = isOrdered ? $"{index + 1}. " : "- ";
_builder.Append(lineItem);
var indent = lineItem.Length;
```

Reply:
Good suggestion, I've changed it to this!